security/fma-kona-asterisc.md

```diff
@@ -0,0 +1,111 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Introduction](#introduction)
+- [Failure Modes and Recovery Paths](#failure-modes-and-recovery-paths)
+  - [[Name of Failure Mode 1]](#name-of-failure-mode-1)
+  - [[Name of Failure Mode 2]](#name-of-failure-mode-2)
+- [Audit Requirements](#audit-requirements)
+- [Action Items](#action-items)
+- [Appendix](#appendix)
+  - [Appendix A: This is a Placeholder Title](#appendix-a-this-is-a-placeholder-title)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Stage 1.4: Kona and Asterisc FMA (Failure Modes and Recovery Path Analysis)
+
+| | |
+|--------|--------------|
+| Author | Andreas Bigger |
+| Created at | 2024-10-17 |
+| Initial Reviewers | *Reviewer Name 1, Reviewer Name 2* |
```

Comment:
nit: initial reviewers are not filled in, they are likely the same as need approval from

Reply:
I left them out since I wasn't sure who would pick up from security and didn't want to forcefully perscribe it to someone, but [it looks like Matt Solomon and Ethnical are the assigned folks now](https://github.com/ethereum-optimism/security-pod/issues/177) so will update.